src/base/bittorrent/trackerentry.h

```diff
@@ -28,17 +28,20 @@
 
 #pragma once
 
-#include <libtorrent/announce_entry.hpp>
-
 #include <QtGlobal>
-
-class QString;
+#include <QString>
+#include <QVector>
 
 namespace BitTorrent
 {
-    class TrackerEntry
+    struct TrackerEntry
     {
-    public:
+        QString url;
+        int tier = 0;
+
+        TrackerEntry() = default;
+        TrackerEntry(const QString &url, int tier = 0);
```

Comment:
I think those constructors are redundant and can be removed.

Reply:
It is for implicitly creating from URL string like it was allowed before (although I'm not sure it's really necessary).

Reply:
>I think those constructors are redundant and can be removed.

Removed.